### DISCUSSION TITLE
Configuring styles in PageImageOptions

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#676] [Q&A] [answered]
Hello Everyone,

is it possible somehow to configure styles in inline images?
For example to set height or width?
I'm using C# code

Thank you in advance for your answers

**Comment by jansenbe** [comment, marked as answer]
@Nevinia210 : at the moment that's not possible, would require an extension of the `PageImageOptions` class. Can you create an issue to track this need or alternatively create a PR for this feature :-)
